addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -1364,11 +1364,12 @@ export function getUrlsInfosInString(string) {
     let infos = [],
         match;
     while ((match = URL_REGEX_WITH_INFOS.exec(string))) {
+        const cleanUrl = match[0].replaceAll('\u200b', '');
```

Comment:
Isn't this gonna fail if the url trully contains a u200b ? I think the proper fix would be that the u200b is removed by the system, just as it is supposed to do. We have code that does that. We should figure out why it doesn't work in this particular case.

Reply:
What use would a 200b be in a URL?
We know why it fails: we change the URL based on the text content in the sanitizer _before_ removing the 200b from the contents (since we sanitize the link first, and then sanitize its contents).
I made the change where I made it because I thought there was no situation in which we'd want to keep a 200b in a URL so I thought it would be safer. But I could instead simply change the sanitizer so the URL changes when we remove the zws (looking up the parent anchor).

Reply:
Sometimes url are paths to a file on a server. The folder name might contain a zero-width space. I have seen multiple cases of this in module folder names while I was working on the Odoo Apps store.